src/driver/postgres/PostgresDriver.ts

```diff
@@ -505,7 +508,38 @@ export class PostgresDriver implements Driver {
             value = DateUtils.stringToSimpleJson(value);
 
         } else if (columnMetadata.type === "cube") {
-            value = value.replace(/[\(\)\s]+/g, "").split(",").map(Number);
+            value = value.replace(/[\(\)\s]+/g, ""); // remove whitespace
+            if (columnMetadata.isArray) {
+                /**
+                 * Strips these groups from `{"(1,2,3)","()",NULL}`:
```

Comment:
Example in this comment should be `{"1,2,3","",NULL}` instead of `{"(1,2,3)","()",NULL}`